source/common/http/access_log/request_info_impl.h

```diff
@@ -54,6 +61,7 @@ struct RequestInfoImpl : public RequestInfo {
   uint64_t response_flags_{};
   Upstream::HostDescriptionConstSharedPtr upstream_host_{};
   bool hc_request_{};
+  std::string downstream_address_;
```

Comment:
I know it's required on the request info. I'm saying to not duplicate it in the connection manager itself. Use the one in the request info. You are copying the same string twice.